### PR TITLE
Translated to Japanese 20200321

### PR DESCRIPTION
**Pull request by TakaoShimizu**
Signed-off-by: TakaoShimizu <takao.shimizu@sysdig.com>
/kind content
/kind transalation
/area documentation

**What this PR does / why we need it**:
Useful for Japanese

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by TakaoShimizu** [comment]
I added dco. What should I do for merge to Master?

File: README.md [hunk 1]
```diff
@@ -35,4 +35,15 @@ To translate docs into Chinese, create Chinese-language Markdown files in the `c
 To translate other parts of the site:
 
 1. See the `languages.zh` block in the site's [`config.yaml`](./config.yaml) configuration file and translate those items from the current English.
-1. See [`i18n/zh.yaml`](./i18n/zh.yaml) for a handful of language snippets that need to be translated from the current English. Translate the `other` block of each term.
\ No newline at end of file
+1. See [`i18n/zh.yaml`](./i18n/zh.yaml) for a handful of language snippets that need to be translated from the current English. Translate the `other` block of each term.
+
+### Japanese
+
+There is currently scaffolding in place for a Japnese translation of the site.
```
Comment (leodido) [inline README.md]:
```suggestion
There is currently scaffolding in place for a Japanese translation of the site.
```

**Comment by TakaoShimizu** [comment]
Already, falco.org has dropdown list for language.
Can we merge to Master?
/assign @fntlnz

**Comment by fntlnz** [comment]
@TakaoShimizu we need you to remove the draft flag from the PR first :)

**Comment by fntlnz** [comment]
I also confirm that once this PR gets merged the dropdown will contain Japanese.

I tested locally too.

image

**Comment by fntlnz** [comment]
@TakaoShimizu I noticed I could do the thing myself and removed the draft since you asked to merge to master and this LGTM

**Pull request merged**